core/Dockerfile

```diff
@@ -1,8 +1,7 @@
 FROM rasa/rasa:1.9.4-full
 EXPOSE 8080
 
-ENV ACTION_SERVER_ENDPOINT=http://actions:8080/webhook
-ENV TRACKER_STORE_ENDPOINT=tracker_store
+ENV TRACKER_STORE_ENDPOINT=tracker-store
```

Comment:
Is this necessary?

Reply:
nop, it should be in docker-compose file actually. moving.